autobot/config_loader.py

```diff
@@ -11,56 +11,105 @@
 import os
 from configparser import ConfigParser
 
+import yaml
 from dotenv.main import dotenv_values
 
 import autobot.config as config
 
 
-class Config:
+class Config(dict):
     """Interact with configuration variables."""
 
     dotenv_path = os.path.join(os.path.abspath(os.path.join(__file__, "..")), ".env")
 
-    ini_parser = ConfigParser()
     ini_path = os.path.join(os.path.abspath(os.path.join(__file__, "..")), "config.ini")
 
-    config = {}
-
-    @classmethod
-    def __init__(cls, **kwargs):
+    def __init__(self, dotenv=None, ini=None, defaults=True, *arg, **kw):
         """Initialize configuration."""
-        cls.load_config(**kwargs)
+        super(Config, self).__init__()
+        self.update(self.load(dotenv=dotenv, ini=ini, defaults=defaults, *arg, **kw))
 
     @classmethod
-    def load_config(cls, **kwargs):
-        """Get autobot configuration values."""
-        cls.config = cls.env_config()
-        cls.ini_parser.optionxform = str
-        cls.ini_parser.read(cls.ini_path)
-        cls.config.update(cls.ini_config())
-        cls.config.update(kwargs)
-        py_config = cls.py_config()
-        defaults = {
-            key: py_config[key] for key in py_config if key not in cls.config.keys()
+    def load(cls, dotenv=None, ini=None, defaults=True, *arg, **kw):
+        """Load configuration."""
+        conf = {}
```

Comment:
```suggestion
        conf = cls()
```

This would have the same effect, but you would end up returning a `Config` object with all the class methods.